lib/vector-buffers/src/variants/disk_v2/writer.rs

```diff
@@ -76,8 +74,6 @@ where
     /// This is nonsensicial because we don't intend to ever support encoding zero-sized types
     /// through the buffer, and the logic we use to count the number of actual events in the buffer
     /// transitively depends on not being able to represent more than one event per encoded byte.
-    ///
-    /// See lemma 4 for more information.
```

Comment:
This is an old comment from when I actually listed out lemmas about the disk buffer design.